### DISCUSSION TITLE
Altri tipi di note

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#588] [Idee e proposte]
Buon pomeriggio Sarebbe possibile nel registro oltre alle note disciplinari altri tipi di note tipo quelle generiche didattiche ecc.

**Comment by trinko** [comment]
Per quello esistono le ANNOTAZIONI.

**Comment by Davide031** [comment]
Si ma non intendo le annotazioni. Parlo di aggiungere le note generiche per ulteriori informazioni oppure si potrebbero aggiumgere con le note generiche mettere la possibilità se mostrarle alla famiglia (quelle generiche)

**Comment by trinko** [comment]
Oltre le annotazioni che si può decidere se notificare ai genitori o no, ci sono anche le OSSERVAZIONI, leggibili sempre dai genitori.

Non mi pare necessario aggiungere altre tipologie.
